### ISSUE TITLE
With a few minor change it run's on a raspberry pi (Debian alike)

### ISSUE THREAD

**Issue by martienpoels-bank2loyalty** [CptWesley/TesserNet#3]
Hi Wesley,

With minimal code changes it run's on a raspberry pi.
On the pi install the tesseract package "apt-get install tesseract-ocr"

In the load section I added support for unix

```csharp
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
            {
                if (Environment.Is64BitOperatingSystem)
                {
                    files = resources.ForPlatform("w64");
                }
                else
                {
                    files = resources.ForPlatform("w86");
                }
            }
            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
            {
                files = resources.ForPlatform("any");
            }
            else
            {
                throw new PlatformNotSupportedException();
	    }
```

Then I changed the UnixTesseractApi.cs file on line 58:
```csharp
           private const string DllPath = "libtesseract.so.4";
```

And that was it. 

If you agree on the changes, please apply them so that I can use the package "out of the box".

Kind greetings and thanks for the good work on the matter,

Martien Poels

**Comment by CptWesley** [comment]
Should be implemented in the new `6.0.0` releases published on NuGet. Feel free to let me know if it worked.

**Comment by martienpoels-bank2loyalty** [comment]
Hi Wesley,

It works like a charm. Now I can share code between windows and raspberry without alterations.
Great job!

Met vriendelijke groet/with kind regards,

Martien Poels
Founder / CTO


[Bank 2 Loyalty]
Contactless technologies bv
Keizersveld 50
5803 AN Venray
The Netherlands
***@***.******@***.***>
www.bank2loyalty.com<http://www.bank2loyalty.com/>
+31 (0) 478 206900<tel:+314787206900>
